storage.go

```diff
@@ -41,6 +42,28 @@ type DiskImageStorageDeviceAttachment struct {
 	*baseStorageDeviceAttachment
 }
 
+// DiskImageCachingMode describes the disk image caching mode.
+//
+// see: https://developer.apple.com/documentation/virtualization/vzdiskimagecachingmode?language=objc
+type DiskImageCachingMode int
+
+const (
+	DiskImageCachingModeAutomatic DiskImageCachingMode = iota
+	DiskImageCachingModeUncached
+	DiskImageCachingModeCached
+)
+
+// DiskImageSynchronizationMode describes the disk image synchronization mode.
+//
+// see: https://developer.apple.com/documentation/virtualization/vzdiskimagesynchronizationmode?language=objc
+type DiskImageSynchronizationMode int
+
+const (
+	DiskImageSynchronizationModeFull  DiskImageSynchronizationMode = 1
+	DiskImageSynchronizationModeFsync DiskImageSynchronizationMode = 2
+	DiskImageSynchronizationModeNone  DiskImageSynchronizationMode = 3
```

Comment:
This is a matter of taste. I like

```suggestion
	DiskImageSynchronizationModeFull  DiskImageSynchronizationMode = 1 + iota
	DiskImageSynchronizationModeFsync DiskImageSynchronizationMode
	DiskImageSynchronizationModeNone  DiskImageSynchronizationMode
```

Reply:
Done